src/main/java/org/example/BillCalculator.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
+
+public class BillCalculator {
+
+    public static double calculate(double valueInvoice, float paymentForService) {
+        return valueInvoice + paymentForService;
+    }
+
+    public static double calculate(double valueInvoice, float paymentForService, double discount) {
+        return valueInvoice - discount + paymentForService;
```

Comment:
Z tym discount to bardziej chodzi o to że ta zmienna powinna przechowywać ile procent zniżki jest ;) i wtedy wyliczasz zniżkę tylko z wartości rachunku